examples/google-cloud-speech/input/cloud_speech_gapic.yaml

```diff
@@ -49,14 +49,6 @@ interfaces:
     total_timeout_millis: 5000000
   methods:
   - name: Recognize
-    flattening:
-      groups:
-      - parameters:
-        - config
-        - audio
-    required_fields:
-    - config
-    - audio
```

Comment:
@landrito @jbolinger Does this look correct? Also, I noticed that [LongRunningRecognize and LongRunningRecognizeRequest](https://github.com/googleapis/gapic-generator-ruby/blob/decbf2396e37976efab86a9b106ea698c873d5d7/examples/google-cloud-speech/input/cloud_speech.proto#L51-L79) are lacking the annotations. Shouldn't they have them, since the attributes (params) are identical?

Reply:
@quartzmo yup, you're correct. The flattening part should be in the annotations. The `required_fields` can be dropped.

Reply:
I added proto annotations and removed YAML entries for LongRunningRecognize.